### PR TITLE
Update to the documentation that was left on my repo for a while && .gitignore file

### PR DESCRIPTION
**Pull request by pieroxy**

### CODE REVIEW

**Comment by montdidier** [comment]
@pieroxy Is this still relevant?

**Comment by pieroxy** [comment]
I believe so. This has been running on my blog for a while now ;-)

These are all small changes and thay all make sense (to me at least)

**Comment by montdidier** [comment]
Ok. I'll spin up a local branch instance and have a quick looks before merging.

**Comment by pieroxy** [comment]
Thanks. Let me know if you find anything suspicious ;-)

File: src/main/webapp/WEB-INF/tags/page.tag [hunk 1]
```diff
@@ -45,6 +71,27 @@
...
+    <c:if test="${displayMode == 'detail'}"><c:choose><c:when test="${not empty blogEntry}"><meta property="article:modified_time" content="<fmt:formatDate pattern="yyyy-MM-dd'T'HH:mm:ssz" value="${blogEntry.lastModified}" />"/></c:when><c:when test="${not empty staticPage}"><meta property="article:modified_time" content="<fmt:formatDate pattern="yyyy-MM-dd'T'HH:mm:ssz" value="${staticPage.lastModified}" />"/></c:when></c:choose></c:if>
+
+
+    <meta name="twitter:site" content="<c:out value="${blog.name}" escapeXml="true"/>" />
+    <meta name="twitter:title" content="${pageTitleAlone}"/>
+    <meta name="twitter:card" content="summary"/>
+    <meta name="twitter:description" content="${truncatedContent}" />
+    <meta name="twitter:image" content="${blogEntry.thumbnailURL}" />
+    <c:if test="${displayMode == 'detail'}"><c:choose><c:when test="${not empty blogEntry}"><meta name="twitter:url" content="${url:rewrite(blogEntry.permalink)}" /></c:when><c:when test="${not empty staticPage}"><meta name="twitter:url" content="${url:rewrite(staticPage.permalink)}" /></c:when></c:choose></c:if>
```
Comment (montdidier) [inline src/main/webapp/WEB-INF/tags/page.tag]: Nice. Support for twitter summaries.

**Comment by montdidier** [comment]
LGTM

**Pull request merged**